presto-main/src/main/java/com/facebook/presto/sql/planner/LogicalPlanner.java

```diff
@@ -514,6 +519,18 @@ private ConnectorTableMetadata createTableMetadata(QualifiedObjectName table, Li
         return new ConnectorTableMetadata(toSchemaTableName(table), columns, properties, comment);
     }
 
+    private RowExpression toRowExpression(Expression expression, SqlPlannerContext context, Analysis analysis)
```

Comment:
Can we share this call from somewhere else? I see multiple copies of it.

Reply:
This call is already shared. TranslateExpressions.toRowExpression is a static function. This is just an extra method in some classes rather than inlining the call. This is just to make it more readable, otherwise we will end up passing 10 arguments in several places

Reply:
I was trying to say that you have many toRowExpression functions declared, they all look same to me. Is it possible have just one declaration of it?